Tests/SentryTests/Helper/SentryFileManagerTests.swift

```diff
@@ -142,13 +145,28 @@ class SentryFileManagerTests: XCTestCase {
         let actualSession = sut.readCurrentSession()
         XCTAssertTrue(expectedSession.distinctId == actualSession?.distinctId)
     }
-    
+
     func testStoreDeleteCurrentSession() {
         sut.storeCurrentSession(SentrySession())
         sut.deleteCurrentSession()
         let actualSession = sut.readCurrentSession()
         XCTAssertNil(actualSession)
     }
+
+    func testStoreAndReadTimestampLastInForeground() {
+        let expectedTimestamp = Date()
+        sut.storeTimestampLast(inForeground: expectedTimestamp)
+        let actualTimestamp = sut.readTimestampLastInForeground()
+        // Why won't compile?
+//        XCTAssertEqual(actualTimestamp, expectedTimestamp, accuracy: 0.001)
```

Comment:
Actually the comparison should work without `accuracy`. I would expect that the storing and reading of the timestamp are exactly the same, but they aren't because the precision of `sentry_fromIso8601String` is only down to [seconds](https://github.com/getsentry/sentry-cocoa/blob/master/Sources/Sentry/NSDate%2BSentryExtras.m#L16). So either we use something different than `sentry_fromIso8601String` to be really accurate or we use a date that has milli- and nanoseconds set to 0 to make this test work. We could use `TestCurrentDateProvider` for this. If we keep using `sentry_fromIso8601String` I would at least add a comment to `storeTimestampLast` about its precision, because this is quite confusing IMO.